CMakeLists.txt

```diff
@@ -303,7 +310,12 @@ target_link_libraries(multiminer mm_gpu_gate ${CMAKE_THREAD_LIBS_INIT} ${CURL_LI
 IF (WIN32)
 	target_link_libraries(multiminer -lws2_32)
 ENDIF()
-target_compile_options(multiminer PRIVATE -O3 -mtune=native -march=native -I.)
+
+IF (WIN32)
+     target_compile_options(a.multiminer PRIVATE -O1 -mtune=native -march=native -I.)
```

Comment:
This commit also uses a.multiminer target. Please create a new branch on your fork without this renaming and make the pull request from there. Thank you.